### PR TITLE
 #24 - change logistics and locations to Porto

### PR DESCRIPTION
**Pull request by filfreire**
(And TBA were applicable)

Results:
Logistics:
screen shot 2018-03-26 at 19 46 40

Location:
screen shot 2018-03-26 at 19 46 19

Venue:
screen shot 2018-03-26 at 19 46 14

@FabioCB @nmatos @filipemcarvalho kindly CR.

related issue: #24

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -210,13 +210,13 @@ findWayWalking: "WALKING"
...
+logisticsMapCenterCoordinates: "41.157944, -8.629107"
+logisticsMapMobileCenterCoordinates: "41.157944, -8.629107"
 logisticsMapAutoDirections: false
 
 # Logistics Direction Details Block
 directionDetailsImage: "direction-details.jpg"
-directionDetailsTitle: "Get around Lviv"
+directionDetailsTitle: "Get around Porto"
 directionDetailsCards:
  - {title: "From the Airport", information: 'Airport is right at the city border. It is easily reachable by car, public transport or taxi. More information is available at <a href="http://lwo.aero/en/transport">airport website</a>.'}
```
Comment (FabioCB) [inline _config.yml]: can we also remove  <a href="http://lwo.aero/en/transport">airport website</a>.'}

File: _config.yml [hunk 2]
```diff
@@ -210,13 +210,13 @@ findWayWalking: "WALKING"
...
  - {title: "Public Transit", information: 'Google Transit is available in Lviv. So you can find your way in Google Maps. Moreover, you can download any of the offline public transit apps available for Lviv.'}
```
Comment (FabioCB) [inline _config.yml]: Change Lviv to Porto
remove: Moreover, you can download any of the offline public transit apps available for Lviv.'

File: _config.yml [hunk 3]
```diff
@@ -90,27 +90,27 @@ rockstarSpeakersButtonText: "See all speakers"
...
-mapCenterCoordinates: "49.842537, 24.025701"
-mapMobileCenterCoordinates: "49.841620, 24.029411"
+eventPlaceCoordinates: "41.157944, -8.629107"
+mapCenterCoordinates: "41.157944, -8.629107"
+mapMobileCenterCoordinates: "41.157944, -8.629107"
 
 # Venue Block
 venueBlockTitle: "Venue"
 venueBigImage: "venue_0.jpg"
 venueSmallImage1: "venue_1.jpg"
```
Comment (FabioCB) [inline _config.yml]: Can we put placeholders instead of this images?

File: _config.yml (hunk 1 above)
Comment (filfreire) [reply _config.yml]: @FabioCB improved

File: _config.yml (hunk 2 above)
Comment (filfreire) [reply _config.yml]: @FabioCB improved

File: _config.yml (hunk 3 above)
Comment (filfreire) [reply _config.yml]: @FabioCB improved

venue_after_fix

**Comment by filfreire** [comment]
@FabioCB improvements finished, kindly re-review

**Pull request merged**